src/microbe_stage/MicrobeCamera.cs

```diff
@@ -56,6 +56,9 @@ public class MicrobeCamera : Camera, IGodotEarlyNodeResolve, ISaveLoadedTracked
     [JsonProperty]
     public float InterpolateZoomSpeed = 0.3f;
 
+    [JsonIgnore]
+    public ShaderMaterial MaterialToUpdate { get { return materialToUpdate; } }
```

Comment:
This can just be `public ShaderMaterial MaterialToUpdate => materialToUpdate`.

Reply:
It could be.
I commented on discord that it would be much better to tell the camera "hey, here's the current light level" rather than allow any other code outside this class to mess with this material object directly.

Reply:
I had no idea that's what you were trying to communicate, I changed it now